hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/checker/RequiredFieldsChecker.groovy

```diff
@@ -151,6 +152,14 @@ class RequiredFieldsChecker extends BaseStaticChecker {
   }
 
   @Override
+  void visitJob(PinotBuildAndPushJob job) {
+    if (job.tableName == null || job.inputPath == null || job.pushLocation == null) {
```

Comment:
Add ` || job.tableName.isEmpty() || job.inputPath.isEmpty() || job.pushLocation.isEmpty()` to the condition?